LiskHQ/lisk-desktop
# Handle transaction dryrun results

### What was the problem?

This PR resolves #4698

### How was it solved?

1. Added an option to the multistep component to be able to navigate to a specific slide. 
2. Use that option to navigate to the first slide when a transaction fail

### How to test?

1. Hardcode the nonce to 0 in TxComposer (this will force a dryrun error)
   <img width="279" alt="image" src="https://github.com/LiskHQ/lisk-desktop/assets/8784876/d41a0cee-2dc8-428d-93a4-1b2f995cae73">
4. Open a dialog to perform a transaction, for example send token
5. Try to send token
6. It should fail and show a retry button in the last step, Click the retry button.
7. You should be navigated to the first step of the form with your pre-filled values

Do the above steps with a few more transaction types, for example register validator, claim rewards, change commission,...
